src/core/utils/compatibility.js

```diff
@@ -23,21 +28,36 @@ export const FACEBOOK_CONTAINER_ADDON_GUID = '@contain-facebook';
 export const FACEBOOK_CONTAINER_DOWNLOAD_URL =
   'https://www.mozilla.org/firefox/facebookcontainer/';
 
-export function getCompatibleVersions({ _log = log, addon, clientApp } = {}) {
+export type GetCompatibleVersionsParams = {|
+  _log?: typeof log,
+  addon: AddonType,
+  clientApp: string,
+  currentVersion: AddonVersionType | null,
+|};
+
+export type CompatibleVersionsType = {|
+  supportsClientApp: boolean,
+  maxVersion: string | null,
+  minVersion: string | null,
+|};
+
+export function getCompatibleVersions({
+  _log = log,
+  addon,
```

Comment:
Why did you keep `addon` here? It is not used anymore, except in a log call.

Reply:
That's the reason I kept it. I assumed we wanted to log the `addon` in that `log` call. I suppose we could use `currentVersion` instead, but that contains less immediately obvious information. 

Do you think we should change this to remove `addon` as an argument?

Reply:
no, I wanted to double-check. It makes sense to keep it in order to get a better log message 👍